internal/querynode/collection.go

```diff
@@ -268,17 +270,18 @@ func (c *Collection) removeVDeltaChannel(channel Channel) {
 }
 
 // setReleaseTime records when collection is released
-func (c *Collection) setReleaseTime(t Timestamp) {
+func (c *Collection) setReleaseTime(t Timestamp, released bool) {
 	c.releaseMu.Lock()
 	defer c.releaseMu.Unlock()
 	c.releaseTime = t
+	c.released = released
 }
 
 // getReleaseTime gets the time when collection is released
-func (c *Collection) getReleaseTime() Timestamp {
+func (c *Collection) getReleaseTime() (Timestamp, bool) {
```

Comment:
Question: Is the returned `release timestamp` preserved for query collection?

Reply:
Currently yes. Keep it in case of query shard shall respect release ts.